src/modules/evbc/client.ts

```diff
@@ -70,10 +71,10 @@ class EVBackendClient {
   // - these shouldn't be callable, until we're successfully connected
   // - it would be nice, if we got an object after successful connection, that contains that
   load_config(name: string) {
-    if (!(name in this._configs)) {
+    if (!(name in this.evbcStore.available_configs)) {
```

Comment:
`in` can be a bit dangerous. It only works on objects. On Arrays or Sets it checks if the Array type has a property.